airflow/providers/databricks/operators/databricks.py

```diff
@@ -250,23 +251,23 @@ class DatabricksSubmitRunOperator(BaseOperator):
     def __init__(
         self,
         *,
-        json=None,
-        spark_jar_task=None,
-        notebook_task=None,
-        spark_python_task=None,
-        spark_submit_task=None,
-        new_cluster=None,
-        existing_cluster_id=None,
-        libraries=None,
-        run_name=None,
-        timeout_seconds=None,
-        databricks_conn_id='databricks_default',
-        polling_period_seconds=30,
-        databricks_retry_limit=3,
-        databricks_retry_delay=1,
-        do_xcom_push=False,
+        json: Optional[Any] = None,
```

Comment:
Should this be `Optional[Dict]`?

Reply:
tried that but below assignment is making mypy check a fail.   
```python
self.json = _deep_string_coerce(self.json) #line 301
```

`_deep_string_coerce` return type is `Union[str, list, dict]` is not matching with the only 'dict' of JSON